fedimint-server/src/net/api.rs

```diff
@@ -296,6 +310,73 @@ impl ConsensusApi {
             .await
             .map_err(|_| ApiError::server_error("Unable send event".to_string()))
     }
+
+    pub async fn build_full_server_status(&self) -> ApiResult<FullStatus> {
+        let our_epoch = self.get_epoch_count().await;
+        // Clone to avoid holding the lock while we calculate things
+        let rejoin_at_epoch = self.rejoin_at_epoch.lock().await.clone();
```

Comment:
There's got to be a better name for `rejoin_at_epoch` because when I'm reading it confuses me (I know you didn't introduce it, just commenting as I'm reading).

Reply:
`rejoin` is a verb